Client/Frontend/Menu/AppMenuConfiguration.swift

```diff
@@ -92,13 +93,25 @@ struct AppMenuConfiguration: MenuConfiguration {
             if #available(iOS 9, *) {
                 menuItems.append(tabState.desktopSite ? AppMenuConfiguration.RequestMobileMenuItem : AppMenuConfiguration.RequestDesktopMenuItem)
             }
+            if NSLocale.currentLocale().localeIdentifier == "zh_CN" {
+                menuItems.append(tabState.isNoImageMode ? AppMenuConfiguration.SelectedNoImageModeItem : AppMenuConfiguration.NoImageModeItem)
+            }
```

Comment:
We're moving towards using static accessor pure functions.

NoImageModeHelper.swift
```swift
class NoImageModeHelper {
    static func isNoImageModeAvailable(state: AppState) -> Bool
    static func isNoImageModeActivated(state: AppState) -> Bool
}
```
See how homepage works with `AppState` in the menu.

In the case of `isNoImageModeAvailable`, I think this should be keyed off a preference, _not_ the locale, as this is definitely not just desirable in Chinese locales. We should probably loop UX in on this, to see how to handle it in non-cn regions (e.g. a "Customize Menu" pref screen and regional defaults).